src/state_manager/mod.rs

```diff
@@ -1047,17 +1047,17 @@ where
         &self,
         mut current: Tipset,
         message: &ChainMessage,
-        look_back_limit: Option<i64>,
-        allow_replaced: Option<bool>,
+        min_epoch: ChainEpoch,
```

Comment:
Perhaps `lookback_max_epoch`. `min` is theoretically correct, but with `lookback` to me the meaning is a bit inverted. Not a huge deal, if you don't agree let's leave it as is (but add `lookback` somewhere).